Server/main.cpp

```diff
@@ -7,7 +7,7 @@
 #include "GameEngine.h"
 #include "Networking/Server.h"
 
-constexpr auto TICKS_PER_SECOND = 10; // How many updates per second.
+constexpr auto TICKS_PER_SECOND = 60; // How many updates per second.
```

Comment:
Curious, why is it 60?

Reply:
Having 60 ticks per second makes it so the movement is smooth. Otherwise, we'd need to add smoothing to game updates on the client. Since the game will run locally anyways, having a lot of updates shouldn't be a problem.